content/blog/2019/root-transition/index.md

```diff
@@ -0,0 +1,150 @@
+---
+title: User guide for extenting Istio root certificate lifetime
+description: The user guide to extend Istio root certificate lifetime.
+publishdate: 2019-06-06
+subtitle:
+attribution: Oliver Liu
+twitter:
+keywords: [security, PKI, certificate, Citadel]
+
+---
+
+## Background
+
+The Istio self-signed certificates have the default lifetime of 1 year.
+If you are using the Istio self-signed certificates,
+please schedule a root transition before it expires.
+An expiration of root certificate may lead to an unexpected cluster-wide outage.
+After the transition, the root certificate will be renewed to have 10 year lifetime.
+
+{{< tip >}}
+We strongly recommend you rotate root keys and root certificates anually as a security best practice.
+We will send out instructions for root key/cert rotation as a follow-up.
+{{< /tip >}}
+
+To evaluate the lifetime remaining for your root certificate, please refer to Step 0 in the following procedure.
```

Comment:
refer to the first step in the procedure below.

and add a link to "procedure below"